src/components/teams/Teams.js

```diff
@@ -294,27 +247,28 @@ export default class Teams extends Component {
   _getTeamsNavigationTab(team) {
     var dangerZone = team.ranking.length - 2;
     var ranking = this._renderOwnTeamPosition(team);
-    if (ranking <= 3) {
-      return ( <NavItem eventKey={team.id}>
-          <div>{team.competition + ' ' + team.teamCode}&nbsp;&nbsp;
-          <span className="label label-as-badge label-success">{ranking}</span></div>
-      </NavItem>
-      );
-    }
-    if (ranking >= dangerZone) {
-      return ( <NavItem eventKey={team.id}>
-          <div>{team.competition + ' ' + team.teamCode}&nbsp;&nbsp;
-          <span className="label label-as-badge label-danger">{ranking}</span></div>
-      </NavItem>
-      );
-    }
+    var style = this._getLabelClassName(ranking,dangerZone);
     return ( <NavItem eventKey={team.id}>
           <div>{team.competition + ' ' + team.teamCode}&nbsp;&nbsp;
-          <span className="label label-as-badge label-default">{ranking}</span></div>
+          <span className={style}>{ranking}</span></div>
       </NavItem>
     );
   }
 
+  _getLabelClassName(ranking,dangerZone) {
+    if (this._isTopper(ranking)) {return 'label label-as-badge label-success';}
+    if (this._isInDegradationZone(ranking,dangerZone)) {return 'label label-as-badge label-danger';}
+    return 'label label-as-badge label-default';
+  }
+
+  _isTopper(ranking) {
+    return ranking <= 3;
+  }
+
+  _isInDegradationZone(ranking,dangerZone) {
+    return ranking >= dangerZone;
+  }
```

Comment:
dit is logica van een "Team", niet logica van het React component.
`team.isTopper()` en `team.isInDegradationZone()` (of een andere naam)

vb die "3" is nu hard coded.
In TeamModel is die 3 ook al gedefinieerd...
Als we dan beslissen, een topper is tussen top4 of top2, dan moeten we meerdere keren "3" gaan zoeken in verschillende files.